CRM/Contact/BAO/Contact/Utils.php

```diff
@@ -257,11 +257,25 @@ public static function createCurrentEmployerRelationship($contactID, $employerID
     }
     if (!is_numeric($employerID)) {
       $dupeIDs = CRM_Contact_BAO_Contact::getDuplicateContacts(['organization_name' => $employerID], 'Organization', 'Unsupervised', [], FALSE);
-      $employerID = (int) (reset($dupeIDs) ?: Contact::create(FALSE)
-        ->setValues([
-          'contact_type' => 'Organization',
-          'organization_name' => $employerID,
-        ])->execute()->first()['id']);
+      if (!empty($dupeIDs)) {
+        $employerID = (int) (reset($dupeIDs));
+      }
+      else {
+        $contact = \Civi\Api4\Contact::get()
```

Comment:
Check permissions need to be disabled for this get to work

Reply:
Fixed, always the implicit TRUE tripping me up.